code/game/objects/items/devices/reverse_bear_trap.dm

```diff
@@ -0,0 +1,129 @@
+/obj/item/device/reverse_bear_trap
+	name = "reverse bear trap"
+	desc = "A horrifying set of shut metal jaws, rigged to a kitchen timer and secured by padlock to a head-mounted clamp."
+	icon_state = "reverse_bear_trap"
+	slot_flags = SLOT_HEAD
+	flags_1 = CONDUCT_1
+	resistance_flags = FIRE_PROOF | UNACIDABLE
+	w_class = WEIGHT_CLASS_NORMAL
+	obj_integrity = 300
+	max_integrity = 300
+	item_state = "rack_parts"
+	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items_righthand.dmi'
+
+	var/ticking = FALSE
+	var/time_left = 60 //seconds remaining until pop
+	var/escape_chance = 0 //chance per "fiddle" to get the trap off your head
+	var/struggling = FALSE
+
+	var/time_since_last_beep = 0
+	var/datum/looping_sound/reverse_bear_trap/soundloop
+	var/datum/looping_sound/reverse_bear_trap_beep/soundloop2
+
+/obj/item/device/reverse_bear_trap/Initialize()
+	. = ..()
+	soundloop = new(list(src))
+	soundloop2 = new(list(src))
+	START_PROCESSING(SSprocessing, src)
+
+/obj/item/device/reverse_bear_trap/Destroy()
+	QDEL_NULL(soundloop)
+	QDEL_NULL(soundloop2)
+	STOP_PROCESSING(SSprocessing, src)
+	return ..()
+
+/obj/item/device/reverse_bear_trap/process()
+	if(!ticking)
+		return
+	time_left--
+	soundloop2.mid_length = max(0.5, time_left - 5) //beepbeepbeepbeepbeep
+	if(!time_left || !isliving(loc))
+		playsound(src, 'sound/machines/microwave/microwave-end.ogg', 100, FALSE)
+		soundloop.stop()
+		soundloop2.stop()
+		to_chat(loc, "<span class='userdanger'>*ding*</span>")
+		addtimer(CALLBACK(src, .proc/snap), 3)
+
+/obj/item/device/reverse_bear_trap/attack_hand(mob/user)
+	if(iscarbon(user))
+		var/mob/living/carbon/C = user
+		if(C.get_item_by_slot(slot_head) == src)
+			if(flags_1 & NODROP_1 && !struggling)
+				struggling = TRUE
+				var/fear_string
+				switch(time_left)
+					if(0 to 5)
+						fear_string = "You agonizingly"
+					if(5 to 20)
+						fear_string = "You desperately"
+					if(20 to 40)
+						fear_string = "You panickedly"
+					if(40 to 50)
+						fear_string = "You shakily"
+					if(50 to 60)
+						fear_string = "You"
```

Comment:
you can cut the "You" out of fear_string and just put it in the message, since all of these start with it